ocs_ci/ocs/pgsql.py

```diff
@@ -461,3 +474,64 @@ def cleanup(self):
             pod.ocp.wait_for_delete(pod.name)
         log.info("Deleting ripsaw configuration")
         RipSaw.cleanup(self)
+
+    def attach_pgsql_pod_to_claim_pvc(
+        self, pvc_objs, postgres_name=None, run_benchmark=True, pgbench_name=None
+    ):
+        """
+        Attaches pgsql pod to created claim PVC
+
+        Args:
+            pvc_objs (list): List of PVC objs which needs to attached to pod
+            postgres_name (str): Name of the postgres pod
+            run_benchmark (bool): On true, runs pgbench benchmark on postgres pod
+            pgbench_name (str): Name of pgbench benchmark
+
+        Returns:
+            pgsql_obj_list (list): List of pod objs created
+
+        """
+        pgsql_obj_list = []
+        for pvc_obj in pvc_objs:
+            try:
+                pgsql_sset = templating.load_yaml(
+                    constants.PGSQL_STATEFULSET_YAML
+                )
+                del pgsql_sset['spec']['volumeClaimTemplates']
+                pgsql_sset['metadata']['name'] = f"{postgres_name}" + f"{pvc_objs.index(pvc_obj)}"
```

Comment:
You are pass the postgres arg name as None, what if someone doesn't pass the postgres name to the function?
The name according to here will be just index.
Please check.

Reply:
done, added as mandatory to send the postgres_name